### PR TITLE
AMLS-5935 | Updated supervision end reasons H1

### PR DESCRIPTION
**Pull request by matthewharris1**
and made characterCounter grey

^

## How Has This Been Tested?
Manually

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [ ] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [?] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/views/include/forms2/textarea.scala.html [hunk 1]
```diff
@@ -54,35 +54,35 @@ <h1>
...
-
-    <span id="with-hint-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
-        You can enter up to @charCount characters
-    </span>
+        <span id="with-hint-info" class="govuk-hint govuk-character-count__message" aria-live="polite">
+            You can enter up to @charCount characters
+        </span>
 
+    </div>
 </div>
```
Comment (ajchilts) [inline app/views/include/forms2/textarea.scala.html:88]: Picky but newline here?

File: public/amls.css [hunk 2]
```diff
@@ -2260,4 +2260,16 @@ textarea:focus {
...
+  color: #6f777b;
+}
+
+.form-field--error .form-control {
+  border: 4px solid #d4351c;
+}
+
+.govuk-textarea.form-control.form-control--full-width {
+  width: 100%
 }
```
Comment (ajchilts) [inline public/amls.css:2275]: ...and here

**Comment by slavvco** [comment]
@ajchilts last commit is fix to not working counter - id's were not matching

**Comment by slavvco** [comment]
@ajchilts this time added some missing styles for character counter error message

**Comment by slavvco** [comment]
@ajchilts all the above new commits were to address different issues with styling of text areas in AMLS-FRONTEND

**Pull request merged**